environment.yml

```
@@ -18,13 +18,15 @@ dependencies:
 - joblib
 - psutil
 - numexpr
-- traits>=4.6.0
-- pyface>=6
-- traitsui>=6
-- imageio>=2.6.1
+- traits
+- pyface>=6,<7
+- traitsui
+- imageio
+- numpydoc
+- tqdm
+- spyder-kernels
```

Not sure if we can do this yet, have you checked with #8009 ?

---

Looks like we probably can since #8009 mentions 1.9.2 and that's what's in conda now

---

... but I guess it should be:
```suggestion
- spyder-kernels>=1.9.2
```

---

Shouldn't `conda` always install the latest available version unless there is a conflict with other (installed) packages? Let's see if CIs come back happy - if we don't test for the behavior in #8009 I will manually test it.

---

I think it does when creating from scratch but I'm not sure what it does if you do `conda env update -f environment.yml` or something similar. But maybe we don't need to worry too much about this case

---

Yes, I would only support creating the env from scratch, everything else will drive us directly into dependency hell.